### ISSUE TITLE
Missing types for `https` object in `server` section.

### ISSUE THREAD

**Issue by elite174** [nksaraf/vinxi#307]
At the moment it's possible to pass `https` object (listhen object for https) to `server` section in config like:

```
export default defineConfig({
  server: {
    /** @ts-ignore vinxi doesn't have types for this, bug */
    https: {
      cert: "./certs/cert.pem",
      key: "./certs/key.pem",
    },
    preset: "bun",
  },
  vite() {
    return viteConfig;
  },
});
```

But there's no clue that you can pass it.

**Comment by Brendonovich** [comment]
This is already fixed on `main`, just waiting on a new release.
https://github.com/nksaraf/vinxi/blob/e9f6a235c67bca6a904929e36ef857e80fb136cd/packages/vinxi/lib/app.js#L15

**Comment by elite174** [comment]
Fixed in 0.13.2
